Add (optional) pgpass steps to postgres installation in docs #3483 (#3603)
mattermost/docs: source/install/install-ubuntu-1604-postgresql.rst

      You might have to install the PostgreSQL client software to use the command.

  The PostgreSQL interactive terminal starts. To exit the PostgreSQL interactive terminal, type ``\q`` and press **Enter**.
  

> 13. (Optional) Postgres recommends the standard way to pass passwords is through a ``.pgpass`` file. It is a special file that HAS to be 0600 in permissions otherwise it doesnt work. One can also pass the location of the file using the ``PGPASS`` env variable.

Reviewer: ```suggestion
13. (Optional) Postgres recommends the standard way to pass passwords is through a ``.pgpass`` file. It is a special file which has to be ``0600`` in permissions otherwise it will not work. One can also pass the location of the file using the ``PGPASS`` env variable.
```

> 13. (Optional) Postgres recommends the standard way to pass passwords is through a ``.pgpass`` file. It is a special file that HAS to be 0600 in permissions otherwise it doesnt work. One can also pass the location of the file using the ``PGPASS`` env variable.

Reviewer 2: As far as I understand using `.pgpass` is just a conveniency to remove the need to type the password at every login. Taking a quick look at the [official documentation](https://www.postgresql.org/docs/9.5/libpq-pgpass.html) it doesn't appear to be explicitly recommended nor a standard way to authenticate. Also the environment variable to pass would be `PGPASSFILE`.

Reviewer: I've edited the content a bit to work with @streamer45's recommendations. @fakela, let me know what you think. If you're happy with the changes, you can commit them. Otherwise, more than happy to talk them over. :)

Author: Looks alot simplified, I will commit them.
 Also are we adding this content on the same section of the other OS's. If yes I would open a seperate PR for them.

> This leaves the security to the user running the application. If you run it as a service, then you need *sudo* permissions to start a service file. As a result only *root* user has read-write access. 

Reviewer 3: Maybe add a link to the official postgres docs like so for more info on the pgpass file as it's not mattermost specific and some details might change in the future. Other than that it looks good to me.

```suggestion
This leaves the security to the user running the application. If you run it as a service, then you need *sudo* permissions to start a service file. As a result only *root* user has read-write access. 

For further info on the pgpass file please refer to `the official PostgreSQL documentation<https://www.postgresql.org/docs/9.4/libpq-pgpass.html>`_.
```

Author: @svelle this is a nice suggestion, will commit the changes

Reviewer 3: Perfect, thanks.
Now if @justinegeffen doesn't have any issues with wording this should be good to merge :)

>   This leaves the security to the user running the application. If you run it as a service, then you need sudo permissions to   start a service file. Therefore, this file can be only root read-write and disabled for everybody else. After all, if you     are root, there's nothing you can't do

Reviewer: ```suggestion
This leaves the security to the user running the application. If you run it as a service, then you need *sudo* permissions to start a service file. As a result only *root* user has read-write access. 
```
